apps/studio/src/schemas/user.ts

```diff
@@ -30,7 +30,10 @@ export const deleteInputSchema = z.object({
   userId: z.string(),
 })
 
-export const deleteOutputSchema = z.boolean()
+export const deleteOutputSchema = z.object({
+  id: z.string(),
+  email: z.string().email(),
+})
```

Comment:
curious - why define an output schema? wouldn't this run the risk of runtime errors next time we expand and return more stuff (and don't adhere to the zod contract here) + additionally, we validate on input because that's comign from an untrusted source

however, our backend is a known trusted source to our frontend, so validation isn't qutie as useful here

Reply:
> next time we expand and return more stuff (and don't adhere to the zod contract here)

IMO being more explicitly typed here let us make better use of typescript and its type-checking + it improves readability on the maintainer on what they can expect from this route